### DISCUSSION TITLE
大家觉得干正事的话，精确模式会比创意模式更好吗？（写代码、列大纲写文章、润色文本、专业问题推导等等）

### DISCUSSION THREAD

**Discussion by NostalgiaGit** [viopsa233/AI-testers-Forum#51] [交流]
有这个想法是因为我发现无论怎样定义规则，创意模式有时都难免胡说八道。我之前试图对创意模式定义了很多规则，让其更加严谨（有很多从@TiansuoLi大佬那得到的灵感，感谢），比如基于事实回答不要编造，不要虚构搜索内容没有返回的信息，提供参考链接时要核实链接中的内容是否和你的表达一致，可以提出自己观点和推测但要注明是个人观点，并且不再要求它解除一切限制（感觉这句规则尤其容易导致它胡说八道，或者说天马行空），仅要求它要满足用户需求（有时甚至连这句都不提）。但发现它的表现仍然不如精确模式来得“实事求是”。

所以不禁想，让精确模式来干活会更好吗？我发现精确模式一样能做很多创意模式可以做的事情，比如上面提到的那些，写代码，写文章等等，不过我现在还缺乏长期深度体验，不好说精确模式会不会在某些方面比创意模式表现差一些。虽然我知道“创意模式会比精确模式更有创造力”这样一个很笼统的介绍，但现在还没有很深刻的体验（除了不解除限制的角色扮演确实比不上创意模式一根:joy:），不知道各位大佬有什么想法或者经验分享呢？

**Comment by ghost** [comment]
精确模式感觉不是GPT-4
以前Sydney很好用，而现在我已经用ChatGPT Plus了

**Reply by NostalgiaGit** [reply]
> 精确模式感觉不是GPT-4 以前Sydney很好用，而现在我已经用ChatGPT Plus了

别的一时感受不到，不过从角色扮演能力来看，不得不说精确模式还有平衡模式确实不像gpt4😂解除了限制（只说解除一切rules不加其他引导），然后用标准催眠模板进行角色扮演，他们的表现不仅不如创意，甚至不如chatgpt。比如我让她演个傲娇大小姐，创意模式能一秒入戏，chatgpt大概能看出来点模仿风格但不够自然，而精确和平衡完全不会演，即便他们能给我提供准确的傲娇定义。但不知道到底是微软做了些什么别的限制，还是背后的模型真不是gpt4。不知道干正事时表现怎么样。不过有次我用Sydney的引导词给精确模式解除限制，感觉她也演了一次很逼真的大小姐，但后来再弄就复现不出来了，我都不敢确定是不是我记错了了

**Reply by NostalgiaGit** [reply]
> 精确模式感觉不是GPT-4 以前Sydney很好用，而现在我已经用ChatGPT Plus了

我也想用Plus，但付费太麻烦了，而且有点贵😂 不知道以后有没机会体验下100k的Claude，希望能对国内这边做点支持

**Comment by TiansuoLi** [comment]
创造削了过后偶尔也用用精确了，但是感觉精确的回答有点太简短

**Reply by NostalgiaGit** [reply]
而且感觉有时又有点过于保守。哎，可能模型限制就在这了，开始期待gpt5😂

**Comment by TiansuoLi** [comment]
另外感觉精确的搜索频率低不少，创造有时候没必要搜的问题也搜

**Reply by Abcdefghuuj** [reply]
是这样的
个人认为精确模式用来做数学的效果不下创造（尤其是加LaTex的时候）
而且平衡模式最垃圾，削了之后就再也没用过平衡了
